composer.json

```diff
@@ -11,14 +11,13 @@
 	"require": {
 		"php": "^7.1",
 		"pd/security-headers": "^1.0",
-		"nette/di": "2.3.*|2.4.*",
-		"nette/application": "2.3.*|2.4.*"
+		"nette/di": "2.4.*|3.0.*",
```

Comment:
```suggestion
		"nette/di": "2.4.11|3.0.*",
```

Reply:
Proc zrovna 2.4.11? Je na to verzi neco zvlastniho?

Reply:
2.4.10 byla jediná pro PHP74 a následující měla nějaké fixy možná přípravu na PHP8/Nette3